sdcm/cluster_k8s/eks.py

```diff
@@ -324,6 +324,29 @@ def destroy(self):
         EksClusterCleanupMixin.destroy(self)
         self.stop_token_update_thread()
 
+    @property
+    def k8s_monitoring_external_ip(self) -> str:
+        if self.USE_MONITORING_EXPOSE_SERVICE:
+            return self.k8s_monitoring_prometheus_expose_service.service_ip
+        else:
+            # Return any IP of the monitoring pod's hosting node
+            for ip_type in ("ExternalIP", "InternalIP"):
+                cmd = (
+                    f"get node --no-headers "
+                    f"-l {self.POOL_LABEL_NAME}={self.MONITORING_POOL_NAME} "
+                    "-o jsonpath='{.items[*].status.addresses[?(@.type==\"" + ip_type + "\")].address}'"
+                )
+                if ip := self.kubectl(cmd, namespace="monitoring").stdout:
+                    return ip
+
+    @property
+    def k8s_monitoring_external_port(self) -> int:
```

Comment:
Instead of doing all this you could just set USE_MONITORING_EXPOSE_SERVICE to true for EksCluster and return ip address and port of the service that is deployed

Reply:
Actually this "USE_MONITORING_EXPOSE_SERVICE" is not needed anymore and must be removed.